checkov/terraform/checks/resource/aws/ECSTaskDefinitionRoleCheck.py

```diff
@@ -0,0 +1,20 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class ECSTaskDefinitionRoleCheck(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure that the Execution Role ARN and the Task Role ARN are different in ECS Task definitions"
+        id = "CKV2_AWS_37"
```

Comment:
```suggestion
        id = "CKV_AWS_245"
```
we only use `CKV2_` for YAML checks